workshops/geoserver/intro/slides/slides.rst

```diff
@@ -131,6 +124,27 @@ The top toolbar contains links to two other pages:
 
 --------------------------------------------------
 
+OSGeo Community 
+===============
+
+The GeoServer website provides:
+
+* stable releases
+* maintenance releases
+
+GeoServer requires Java 8 to run and includes its own application server.
+
+This workshop both GeoServer and GeoSever Extensions for importer, ysld and wps.
+
+Presenter notes
+---------------
+
+GeoServer, being a Java application, typically requires a Java Runtime Environment (JRE) as well as an application server in order to function.
+
+Each GeoServer release lasts a year, for the first six months dot releases provide improvements and fixes, for the last six months maintenance releases provides fixes only.
```

Comment:
Rephrase the sentence to say something other than "dot releases" which is unclear even to me. Also releases don't really provide improvements and fixes, they contain them.